### ISSUE TITLE
choose a favicon.

### ISSUE THREAD

**Issue by toger5** [toger5/godotdevs-Forum#2]
As I'm not toatally happy with the favicon.
We may could start a contest on the forum or just create some more to see which one does fit best.

**Comment by ghost** [comment]
I like the favicon of godotengine.org. It should be okay to use that, and users can spot easily the "official" Godot tabs in their browser.

**Comment by KioriSun** [comment]
@DocAngelo i don't think we can use that, we can use something similar, but i'm not sure copying anything from the main engine website would be o.k. Even considering how open everything is.
